test/fixtures/controller.line/cubicInterpolationMode/scriptable.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+	config: {
+		type: 'line',
+		data: {
+			labels: [0, 1, 2, 3, 4, 5],
+			datasets: [
+				{
+					// option in dataset
+					data: [0, 4, 2, 6, 4, 8],
+					borderColor: '#ff0000',
+					cubicInterpolationMode: function(ctx) {
+						var mode = ctx.datasetIndex === 0 ? 'monotone' : 'default';
```

Comment:
We should return the result directly, without the temporary variable (`mode`):

`return ctx.datasetIndex === 0 ? 'monotone' : 'default';`